cleanlab/filter.py

```diff
@@ -26,11 +26,11 @@
 import numpy as np
```

Comment:
Could you add one more unit test that runs find_label_issues with n_jobs = 1 vs 2 vs None and verifies the returned label issues match?  This is sort of being minimally tested here:
https://github.com/cleanlab/cleanlab/blob/master/tests/test_filter_count.py#L866

but I think another test of this would be good. You could easily add it here in this method:
https://github.com/cleanlab/cleanlab/blob/master/tests/test_filter_count.py#L698